### PR TITLE
doc: add json entry for latest sec vuln

### PR DESCRIPTION
**Pull request by mhdawson**

### CODE REVIEW

File: vuln/core/42.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "cve": [
+    "CVE-2017-14849"
+  ],
+  "vulnerable": "8.5.0",
+  "patched": "^8.6.0",
+  "ref": "https://nodejs.org/en/blog/vulnerability/september-2017-path-validation/",
+  "overview": " Node.js version 8.5.0 included a change which caused a security vulnerability in the checks on paths made by some community modules. As a result, an attacker may be able to access file system paths other than those intended. \n\n"
```
Comment (cjihrig) [inline vuln/core/42.json]: There is a leading space here and a trailing space. Also, are the `\n`s significant?

**Comment by mhdawson** [comment]
I put the \ns in as they were in some other ones that I used as an example.  @sam-github are they needed ?

**Comment by mhdawson** [comment]
Pushed change to address comments

**Comment by mhdawson** [comment]
Landed as 24bc2f994d4cd0568818827eb96f99848b4d1d4d

**Pull request closed without merging**